packages/lz4_chans/lz4_chans.3.0.0/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/lz4-chans"
+bug-reports: "https://github.com/UnixJunkie/lz4-chans/issues"
+dev-repo: "git+https://github.com/UnixJunkie/lz4-chans.git"
+license: "BSD-3-Clause"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test"] {with-test}
```

Comment:
```suggestion
  ["dune" "exec" "-p" name "src/test.exe"] {with-test}
```
`dune build` without `-p` or `--profile=release` promotes some warnings as errors. This should be avoided in releases.